qcodes/instrument_drivers/Aviosys/IP_Power_9258S.py

```diff
@@ -0,0 +1,92 @@
+from qcodes import Instrument, InstrumentChannel, ChannelList
+from qcodes.utils.validators import Enum
+from qcodes.utils.helpers import create_on_off_val_mapping
+import urllib
+
+
+class PowerChannel(InstrumentChannel):
+    """
+    Channel class for a socket on the Aviosys IP Power 9258S.
+    Args:
+        parent (Instrument): Parent instrument.
+        name (str): Channel name.
+        channel (str): Alphabetic channel id.
+    Attributes:
+        channel (str): Alphabetic channel id.
+        channel_id (int): Numeric channel id.
+    """
+
+    _channel_values = Enum('A', 'B', 'C', 'D')
+    _channel_ids = {'A': 1, 'B': 2, 'C': 3, 'D': 4}
+
+    def __init__(self, parent, name, channel):
+
+        super().__init__(parent, name)
+
+        # validate the channel id
+        self._channel_values.validate(channel)
+        self.channel = channel
```

Comment:
for improving the clarity, could you rename the `channel` attribute to `_id_name` and `channel_id` attribute to `_id_number` (also making them private)? because there are: channel objects, channel name of the device ('A'), channel name as a submodule of the driver class ('A'), name of the channel object ('ChanA' as the driver code below suggests), etc. It's important to not confuse those, and names is a good way of achieving that.

The confusion is already here within this PR. In the driver code below, the term `channel_id` was used locally to refer to `_id_name` while here, in the code of the channel, `channel_id` is actually `_id_number`. It's not a runtime problem, but it's not good for readability (hence debugging, extending, etc.).